### ISSUE TITLE
Opening GoW II MDL_ files causes an error to appear

### ISSUE THREAD

**Issue by Electric-R4nger** [mogaika/god_of_war_browser#2]
This is an amazing and underappreciated tool, however after the update to 1.103 opening any GoW II MDL_ file causes an error message in the console log.

```
2019/12/28 17:42:13 err
goroutine 36 [running]:
runtime/debug.Stack(0x4d8677, 0x1203e060, 0x2)
        C:/go/src/runtime/debug/stack.go:24 +0x83
runtime/debug.PrintStack()
        C:/go/src/runtime/debug/stack.go:16 +0x1a
github.com/gorilla/handlers.recoveryHandler.log(0x8eb8e0, 0x12054960, 0x0, 0x0, 0x1, 0x12006198, 0x1, 0x1)
        c:/gopath/src/github.com/gorilla/handlers/recovery.go:89 +0x53
github.com/gorilla/handlers.recoveryHandler.ServeHTTP.func1(0x32930068, 0x1203a160, 0x8eb8e0, 0x12054960, 0x0, 0x0, 0x1)
        c:/gopath/src/github.com/gorilla/handlers/recovery.go:74 +0xa2
panic(0x7ddb40, 0x8e55f8)
        C:/go/src/runtime/panic.go:679 +0x162
github.com/mogaika/god_of_war_browser/pack/wad/mesh.NewFromData(0x12184000, 0x2f3a0, 0x2f3a0, 0x12006180, 0x12006158, 0x0, 0x0)
        c:/gopath/src/github.com/mogaika/god_of_war_browser/pack/wad/mesh/mesh.go:121 +0xb4
github.com/mogaika/god_of_war_browser/pack/wad/mesh.init.0.func2(0x12050220, 0x0, 0x0, 0x0, 0x0)
        c:/gopath/src/github.com/mogaika/god_of_war_browser/pack/wad/mesh/mesh.go:150 +0x267
github.com/mogaika/god_of_war_browser/pack/wad.(*Wad).CallHandler(0x1203e0c0, 0x2e, 0x1203e9c0, 0x0, 0x10004, 0x8eb840, 0x12006130)
        c:/gopath/src/github.com/mogaika/god_of_war_browser/pack/wad/wad.go:90 +0xa3
github.com/mogaika/god_of_war_browser/pack/wad.(*Wad).GetInstanceFromNode(0x1203e0c0, 0x2e, 0x1203e9c0, 0x0, 0x10004, 0x8eb840, 0x12006130)
        c:/gopath/src/github.com/mogaika/god_of_war_browser/pack/wad/wad.go:109 +0x70
github.com/mogaika/god_of_war_browser/pack/wad/mdl.(*Model).Marshal(0x1201e230, 0x12050960, 0x12050960, 0x1201e230, 0x2000f, 0x0)
        c:/gopath/src/github.com/mogaika/god_of_war_browser/pack/wad/mdl/mdl.go:61 +0xd5
github.com/mogaika/god_of_war_browser/pack/wad.(*Wad).WebHandlerForNodeByTagId(0x1203e0c0, 0x32930068, 0x1203a160, 0x62, 0x0, 0x1203e0c0)
        c:/gopath/src/github.com/mogaika/god_of_war_browser/pack/wad/webhandlers.go:23 +0x1ee
github.com/mogaika/god_of_war_browser/web.HandlerAjaxPackFileParam(0x32930068, 0x1203a160, 0x12090380)
        c:/gopath/src/github.com/mogaika/god_of_war_browser/web/handlers.go:62 +0x246
net/http.HandlerFunc.ServeHTTP(0x87030c, 0x32930068, 0x1203a160, 0x12090380)
        C:/go/src/net/http/server.go:2007 +0x34
github.com/gorilla/mux.(*Router).ServeHTTP(0x12054960, 0x32930068, 0x1203a160, 0x12072380)
        c:/gopath/src/github.com/gorilla/mux/mux.go:210 +0xbb
github.com/gorilla/handlers.recoveryHandler.ServeHTTP(0x8eb8e0, 0x12054960, 0x0, 0x0, 0x1, 0x32930068, 0x1203a160, 0x12072380)
        c:/gopath/src/github.com/gorilla/handlers/recovery.go:78 +0x7c
github.com/gorilla/handlers.loggingHandler.ServeHTTP(0x8ec320, 0x120420d8, 0x8eb8a0, 0x120b3760, 0x87023c, 0x8f03c0, 0x12102000, 0x12072380)
        c:/gopath/src/github.com/gorilla/handlers/logging.go:45 +0x1cc
net/http.serverHandler.ServeHTTP(0x120607e0, 0x8f03c0, 0x12102000, 0x12072380)
        C:/go/src/net/http/server.go:2802 +0x84
net/http.(*conn).serve(0x120545a0, 0x8f0d80, 0x120b2f00)
        C:/go/src/net/http/server.go:1890 +0x7ad
created by net/http.(*Server).Serve
        C:/go/src/net/http/server.go:2927 +0x311
127.0.0.1 - - [28/Dec/2019:17:42:12 +0100] "GET /json/pack/PART1-0-R_HERO01.WAD/98 HTTP/1.1" 500 0
```

Any other file for GoW II opens just fine, and GoW I MDL_ files open fine also.

**Comment by mogaika** [comment]
Thanks for the report! Really appreciate this.
Recently I implemented gow1 mesh fields parsing related to mesh instansing and joints mapping, looks like they incompatible with gow2 version. Will check soon

**Comment by mogaika** [comment]
Must be fixed in 1063656ffa5ca7992f31b8738d5e9d2dc7c42f3d
